app/notifiers/publishing_api_notifier.rb

```diff
@@ -0,0 +1,14 @@
+class PublishingApiNotifier
+  def self.publish
+    new.publish
```

Comment:
I tend to favour a cleaner internal API between objects, particularly for fire-and-forget, 'command'-like method invocations like this one (i.e. - we don't really care about the return value, we don't get an instance of a domain object back). The method definition is cheap and obvious, and spares the unnecessary extra step of calling `new` from the callsite.